### PR TITLE
Add md file for command get_raw_transaction

### PR DESCRIPTION
**Pull request by hrmhatef**
Please review the file.

@smk762

### CODE REVIEW

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method returns hexadecimal encoded signed transactions.
+
+## Arguments
+|Structure|Type|Description|
+|:-------:|:---:|:--------:|
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:6]: Not sure if it matters for vuepress, but normally we don't have the `:` for table formating, and try to align the `|` so it's a bit easier to read. eg - 

```


|Param    | Type   | Description           |
|---------|--------|-----------------------|
|field    | string | detail about field    |

```

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# get_raw_transaction
+The  `get_raw_transaction`  method returns hexadecimal encoded signed transactions.
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:2]: Slightly more verbose ELI5 version -

`The `get_raw_transaction` method takes a coin and confirmed txid as input, and returns the returns its full signed raw transaction hex.`

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+{
+  "mmrpc":"2.0",
+  "result":{
+  "tx_hex":"0400008085202f89025655b6fec358091a4a6b34107e69b10bd7660056d8f2a1e5f8eef0db6aec960100000000494830450221008c89db5e2d93d7674fe152e37344dfd24a0b1d4d382a7e0bcfc5d8190a141d72022050ce4ef929429e7e1a6c4ebd3f72a1a2aa25da1e0df65553a2c657658077ed1d01feffffff79cc137b70c39c9c7c2b9230c818ec684ffe731bf1ae821f91ba9d3e526f55f00000000049483045022100868c71f4a8e1452a3bc8b1d053a846959ab7df63fb0d147e9173f69818bbb1f3022060c7e045a34cf6af61bc3a74dc2db7b8bfa4949bc5919acceed40fc07d8706d201feffffff0240043a0000000000232102afdbba3e3c90db5f0f4064118f79cf308f926c68afd64ea7afc930975663e4c4ac201efc01000000001976a914347f2aedf63bac168c2cc4f075a2850435e20ac188ac96d3c96036dd0e000000000000000000000000"
+  },
+  "id":0
+}
+```
+
+### Error response:
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:41]: Can add another error response like 

{
	"mmrpc": "2.0",
	"error": "Transport error: rpc_clients:1770] JsonRpcError { client_info: \"coin: RICK\", request: JsonRpcRequest { jsonrpc: \"2.0\", id: \"49\", method: \"blockchain.transaction.get\", params: [String(\"889360b0225b4e05fa13643e2e306c8eb5c52fa611615dfd30195089010b1c7b\"), Bool(false)] }, error: Response(electrum1.cipig.net:10017, Object({\"code\": Number(2), \"message\": String(\"daemon error: DaemonError({'code': -5, 'message': 'No information available about transaction'})\")})) }",
	"error_path": "utxo_common",
	"error_trace": "utxo_common:1810]",
	"error_type": "Transport",
	"error_data": "rpc_clients:1770] JsonRpcError { client_info: \"coin: RICK\", request: JsonRpcRequest { jsonrpc: \"2.0\", id: \"49\", method: \"blockchain.transaction.get\", params: [String(\"889360b0225b4e05fa13643e2e306c8eb5c52fa611615dfd30195089010b1c7b\"), Bool(false)] }, error: Response(electrum1.cipig.net:10017, Object({\"code\": Number(2), \"message\": String(\"daemon error: DaemonError({'code': -5, 'message': 'No information available about transaction'})\")})) }",
	"id": 1
}


{
	"mmrpc": "2.0",
	"error": "Invalid  hash: Invalid character 'x' at position 56",
	"error_path": "utxo_common",
	"error_trace": "utxo_common:1809]",
	"error_type": "InvalidHashError",
	"error_data": "Invalid character 'x' at position 56",
	"id": 1
}

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+|coin|string|the name of the coin the user desires to request for the transaction |
+|tx_hash|string|hash of the transaction|
+
+## Response
+|Structure|Type|Description|
+|:--------:|:----:|:------:|
+|tx_hex|BytesJson|raw bytes of signed transaction|
+
+
+## Examples:
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:16]: Use `#### :pushpin: Examples`

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+### Request
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:18]: Wrap responses with 

```

<div style="margin-top: 0.5rem;">
<collapse-text hidden title="Response">
```
Responses go here

```
</collapse-text>
</div>```

This adds a button to show/hide he responses like
![image](https://user-images.githubusercontent.com/35845239/162945548-eb129952-5705-4d1a-83aa-8ae2694980cf.png)

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  "error_trace": "lp_coins:2234] lp_coins:2156]",
+  "error_type": "NoSuchCoin",
+  "error_data": {
+    "coin": "KMD"
+  },
+  "id": 0
+}
+```
+
+## Error Types
```
Comment (smk762) [inline docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:56]: We dont generally add this table, but I like it, it's a good idea :)

**Comment by smk762** [comment]
Thanks for the PR!
You can also add an entry for the doc in the sidebar like https://github.com/KomodoPlatform/developer-docs/pull/350/files#diff-e8c3c297fe36868e8b4c9daf0f5db3e8bd9c6593e3b817ce890852971cdc0b0dL340

File: docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md (hunk 6 above)
Comment (hrmhatef) [reply docs/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md:78]: Thank you for your comments.

It's done. please review it.

**Comment by hrmhatef** [comment]
I will do it for the next time.
